<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Amplifier</class>
 <widget class="QMainWindow" name="Amplifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amplifier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Fender '57 Deluxe</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fender '59 Bassman</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fender '57 Champ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fender '65 Deluxe Reverb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fender '65 Princeton</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fender '65 Twin Reverb</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fender Super-Sonic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>British 60's</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>British 70's</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>British 80's</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>American 90's</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Metal 2000</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QDial" name="dial"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QDial" name="dial_2"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QDial" name="dial_3"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_3"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QDial" name="dial_4"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_4"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QDial" name="dial_5"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_5"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QDial" name="dial_6"/>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_6"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="setButton">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
